File: hello-python/target.py
#!/usr/bin/env python

"""
@author		:  Rajan Khullar
@created	:  03/14/16
@updated	:  03/14/16
"""

import sys

if __name__ == '__main__':

	# check for correct number of args
	if len(sys.argv) < 3:
		print 'missing parameters'
		sys.exit(1)

	# pass message
	msg = sys.argv[1]

	# pass number
	try:
		n = int(sys.argv[2])
	except ValueError:
		print 'invalid number'
		sys.exit(1)

	# print message specified number of times
	for i in range(n):
		print msg

File: hello-java/target.java
/*
 * @author  :  Rajan Khullar
 * @created :  03/14/16
 * @updated :  03/14/16
 */

public class target
{
	private static String msg;
	private static int n, x;

	public static void main(String[] args)
	{
		// check for correct number of args
		if(args.length < 2)
		{
			System.out.println("missing parameters");
			System.exit(1);
		}

		// pass message
		msg = args[0];

		// pass number
		try
		{
			n = Integer.parseInt(args[1]);
		} catch (NumberFormatException ex)
		{
			System.out.println("invalid number");
			System.exit(1);
		}

		// print message specified number of times
		for(x=0; x<n; x++)
			System.out.println(msg);
	}
}
